### PR TITLE
add monitoring labels to ci, presubmit jobs

### PR DESCRIPTION
**Pull request by yt3liu**
**What this PR does, why we need it**:

Add the monitoring labels to ci, presubmit jobs 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes to reviewers**:

**User-visible changes in this PR**:

/assign @srinivashegde86 
/assign @steuhs

### CODE REVIEW

File: ci/prow/make_config.go [hunk 1]
```diff
@@ -481,6 +489,10 @@ func generatePresubmit(title string, repoName string, presubmitConfig yaml.MapSl
 			if len(data.Base.Args) == 0 {
 				data.Base.Args = []string{"--" + jobName}
 			}
+
```
Comment (adrcunha) [inline ci/prow/make_config.go]:
```suggestion
```

File: ci/prow/periodic_config.go [hunk 2]
```diff
@@ -55,13 +55,23 @@ type periodicJobTemplateData struct {
 	PeriodicCommand []string
 }
 
+// generatePeriodicJobName generate the job name from the suffix
+func generatePeriodicJobName(repoName, jobNameSuffix string) string {
```
Comment (adrcunha) [inline ci/prow/periodic_config.go]: Unused?

File: ci/prow/make_config.go (hunk 1 above)
Comment (yt3liu) [reply ci/prow/make_config.go]: Done

File: ci/prow/periodic_config.go (hunk 2 above)
Comment (yt3liu) [reply ci/prow/periodic_config.go]: Removed. Thanks for catching it.

**Pull request merged**